Code/GraphMol/SmilesParse/CanonicalizeStereoGroups.cpp

```diff
@@ -0,0 +1,1012 @@
+
+//  Copyright (C) 2001-2021 Greg Landrum and other RDKit contributors
```

Comment:
Please just make this 2024. And it doesn't need to be my name

Reply:
ok